lsp-semantic-tokens.el

```diff
@@ -378,6 +378,23 @@ IS-RANGE-PROVIDER is non-nil when server supports range requests."
                        (lsp-warn "No face has been associated to the %s '%s': consider adding a corresponding definition to %s"
                                  category id varname)) maybe-face)) identifiers)))
 
+(defun lsp-semantic-tokens--replace-alist-values (a b)
+  "Replace alist A values with B ones if exists."
```

Comment:
`if exists` sounds strange; not sure how to formulate this correctly without blowing up the docstring's length though. Maybe "if present" or "where available" would be slightly better?
Apart from that, the changes look good to me and seem to work as intended (tested with clojure-lsp)